src/main/java/racingcar/domain/Name.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain;
+
+public class Name {
+
+    private static final int NAME_LOWER_BOUND = 1;
+    private static final int NAME_UPPER_BOUND = 5;
+    private static final String INVALID_NAME_LENGTH_MESSAGE = "차의 이름은 1자 이상 ~ 5자 이하여야 합니다.";
+
+    private final String value;
+
+    public Name(final String name) {
+        validate(name);
+        this.value = name;
+    }
+
+    private void validate(String name) {
+        if (name.length() < NAME_LOWER_BOUND || NAME_UPPER_BOUND < name.length()) {
+            throw new IllegalArgumentException(INVALID_NAME_LENGTH_MESSAGE);
```

Comment:
어떤 오류사항인지 자세히 메시지를 작성해 주신 부분은 좋습니다 👍 다만 실제로 인자를 받은 name 까지 표현해주어 다른 개발자(여기엔 본인도 포함됩니다)에게 오류의 맥락을 풍부하게 해주는 것도 좋아요~~! 

```suggestion
            throw new IllegalArgumentException(INVALID_NAME_LENGTH_MESSAGE + " 입력된 차 이름 : " + name); 
```

Reply:
좋은 의견 감사합니다 👍👍 어떤 필드에서 오류가 발생했고, 어떤 입력을 했는지 꼼꼼하게 작성하도록 해야겠습니다!